### DISCUSSION TITLE
How can I deploy pytest as a test software

### DISCUSSION THREAD

**Discussion by jefflieu** [pytest-dev/pytest#11729] [General]
I have developed a set of test to test a bunch of binary applications. 
How can I package and install these pytest .py files for an operator to run on a machine that doesn't  have access to the source repository?
Is there a standard practice to deploy the pytest as a test application?
Thanks

**Comment by RonnyPfannschmidt** [comment]
In such cases its common to either ship the test suite with the software itself, or to package it as own python package

There's no common standard to do full shipping

In case the test suite is doing black box testing it may work out to just ship it in a container
